azure-quantum/tests/unit/common.py

```diff
@@ -57,6 +58,9 @@ def __init__(self, method_name):
         self._location = os.environ.get("AZURE_QUANTUM_WORKSPACE_LOCATION", os.environ.get("LOCATION", LOCATION))
         self._user_agent = os.environ.get("AZURE_QUANTUM_PYTHON_APPID")
 
+        # needed to make sure the Default Credential will work
+        os.environ["LOCATION"] = self._location
```

Comment:
Is this change related?

Reply:
Not directly related, but helps with the recording of unit tests following the readme.
Since it's just a unit test, it has no effect on the release and I would like to keep it.